src/Utils.js

```diff
@@ -157,11 +158,11 @@ export function coercePropValue(propValue) {
 
   // if propValue includes quotes, it should be
   // treated as a string
-  if (propValue.search(/"/) !== -1) {
-    return propValue.replace(/"/g, '');
+  if (/^(['"]).*\1$/.test(propValue.trim())) {
```

Comment:
Although I don't know if that's actually faster, so I would go with whatever is clearest (which is likely not the regex, lol).